beeswax/openrtb/seat_constraints.proto

```diff
@@ -0,0 +1,15 @@
+// Copyright 2019, Beeswax.IO Inc.
+// Author gil@beeswax.com
+//
+// Constraints on an exchange seat for a BidRequest 
+
+syntax="proto2";
+package openrtb;
+
+option java_package = "com.beeswax.openrtb";
+option cc_enable_arenas = true;
+
+// Next available tag: 2
+message SeatConstraints {
+  optional int64 bid_floor_micros = 1 [default = 0];
```

Comment:
maybe I'm confused about how protos work, but shouldn't there be an account id in here somewhere?

Reply:
@hostdp6 account id is the key to the map https://github.com/BeeswaxIO/beeswax-api/blob/bf88c2cbaed139ae4c5be0ff7d3ead215b3586d9/beeswax/openrtb/extension.proto#L463

The extension `seat_constraints` containts:
1. A `default_seat_constraint`of type `SeatConstraint` and a map from `account_id` to `SeatConstraint`

Reply:
i see, I guess I was expecting to see it explicitly.